### ISSUE TITLE
Keep original manifest filename, support list of files

### ISSUE THREAD

**Issue by ComeBurguburu** [arttor/helmify#100]
Hi,
I want to use helmify on many files and keeping original name and mutualise values.yaml

Here is how I run it with bash

```bash
rm -rf pypi-server

for f in $(ls pypi-server-src);
do 
  cat pypi-server-src/$f | ./helmify pypi-server;
  mv pypi-server/templates/pyserver.yaml pypi-server/templates/$f;
  cat pypi-server/values.yaml >> pypi-server/values-all.yaml;
done
````

**Comment by arttor** [comment]
Hi @ComeBurguburu. Can you please elaborate on which part is missing in helmify for your use-case?

As i can see, the bash script handles manifests one-by-one which will not work, because:
- it will overwrite chart every time and may lose data from previous manifests
- it will not preserve correct links between chart objects

This means that all app manifests should be passed to helmify at once. 
Currently, helmify supports only stdin as a source. You can use this script to pass all manifest from the given directory:
```shell
awk 'FNR==1 && NR!=1  {print "---"}{print}'  pypi-server-src/*.yaml | helmify pypi-server
```
The script reads all yamls from dir (script can be modified to read recursively and to use other file extensions), separates yaml objects with `---` separator and pass everything to helmify at once.


The best option for overriding values.yaml is to use native helm features for overriding files (`-f` ) or concrete params (`--set`):
```shell
helm install -f myvalues.yaml  --set foo=bar
```

**Comment by ComeBurguburu** [comment]
Hi arttor,
Thanks for this amazing tools

I find two issues: 
- helmify can't have a folder as source (I want to keep one file per object)
- service and ingress file generated are named after the chart name

**Comment by arttor** [comment]
got it. in this case, it makes sense to adjust the issue, to support input from the filesystem.

**Comment by arttor** [comment]
Hi @ComeBurguburu, the feature implemented in [v0.4.0 release](https://github.com/arttor/helmify/releases/tag/v0.4.0). Check it out!
